packages/libs/error-stack/src/compat/mod.rs

```diff
@@ -0,0 +1,23 @@
+#[cfg(feature = "anyhow")]
+mod anyhow;
+#[cfg(feature = "eyre")]
+mod eyre;
+
+/// Compatibility trait to convert from external libraries to [`Report`].
+///
+/// [`Report`]: crate::Report
+#[cfg(any(feature = "anyhow", feature = "eyre"))]
+pub trait Compat: Sized {
```

Comment:
```suggestion
pub trait IntoReportCompat: Sized {
```

I think this name is less confusing